_docs/installing/debian.md

```diff
@@ -3,31 +3,35 @@ title: Debian
 category: Installing the bot
 order: 5
 ---
-
 <img class="doc-img" src="{{ site.baseurl }}/images/debian.png" alt="debian" style="width: 75px; float: right;"/>
 
 Installing the bot on Debian is similar to [Ubuntu](/installing/ubuntu), but requires some additional system dependencies obtained through `apt`.
 
-~~~ bash
+~~~bash
 # Update system repositories
 sudo apt-get update -y
 sudo apt-get upgrade -y
 
 # Install dependencies
 sudo apt-get install git libopus-dev libffi-dev libsodium-dev ffmpeg -y
-sudo apt-get install build-essential libncursesw5-dev libgdbm-dev libc6-dev zlib1g-dev libsqlite3-dev tk-dev libssl-dev openssl -y
+sudo apt-get install build-essential libncursesw5-dev libgdbm-dev libc6-dev zlib1g-dev libsqlite3-dev tk-dev libssl-dev openssl libncurses5-dev libnss3-dev libreadline-dev wget libbz2-dev -y
```

Comment:
Double check if Curl comes with Debian, and verify if all of these dependencies are necessary. Notably libncurses5-dev and libncursesw5-dev likely share libraries, and we likely don't need both. We shouldn't unnecessarily install new dependencies on a users system and add bloat when it's not needed.

Reply:
Curl added incase. Removed libreadline-dev.

Reply:
Wait a sec, why do we need curl actually? This page only uses wget.

Reply:
> Double check if Curl comes with Debian
I'm more so meaning we shouldn't install wget on a users system if the system already has Curl.